client/src/App.js

```diff
@@ -1,12 +1,118 @@
 import React from 'react';
 import './App.css';
+import { Switch, Route } from 'react-router-dom';
+
+// ---------------- imports all the other components
+
+import About from './About';
+import Navbar from './Navbar';
+import Portfolio from './Portfolio';
+import Interests from './Interests';
+import Work from './Work'
+import Footer from './Footer'
+import Header from './Header'
+import Modal from './Modal.js';
+import Home from './Home'
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      modalDisplayed: false,
+      language: 'EN',
+      home: "Home",
+      about: "About Me",
+      portfolio: 'Portfolio',
+      work: "Work History",
+      interests: 'Interests',
+    }
+  }
+
+  // opens modal
+
+  modalOpen = () => {
+    this.setState({
+      modalDisplayed: !this.state.modalDisplayed
+    })
+  }
+
+  // three functions to switch among languages (I tried to make a single changeLanguage function, but my if-conditions did not work)
+
+  changetoFrench = () => {
+    this.setState({
+      language: 'FR',
+      home: "Accueil",
+      about: 'À propos de moi',
+      work: 'Travail',
+      interests: 'Mes intérêts'
+    })
+  }
```

Comment:
Good! This is a really cool feature and great use of react to dynamically switch the content of your page.